backend/internal/storage/postgres/works.go

```diff
@@ -0,0 +1,144 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/CamPlume1/khoury-classroom/internal/models"
+	"github.com/jackc/pgx/v5"
+)
+
+const DesiredFields = `
```

Comment:
Does this need to be public?

Reply:
its golang const naming conventions, must be camel case